core/trino-main/src/main/java/io/trino/execution/QueryManagerConfig.java

```diff
@@ -95,6 +95,7 @@
     private DataSize faultTolerantExecutionTaskDescriptorStorageMaxMemory = DataSize.ofBytes(Math.round(AVAILABLE_HEAP_MEMORY * 0.15));
     private int faultTolerantExecutionPartitionCount = 50;
     private boolean faultTolerantPreserveInputPartitionsInWriteStage = true;
+    private boolean faultTolerantExecutionEventDriverSchedulerEnabled = true;
```

Comment:
Not sure about this one. Do we can get some adoption, while keeping it `false` for a release or two?

Reply:
I ran multiple rounds of full scale testing and it seems to work fine. I would probably leave it on by default while leaving the old implementation as a fallback option if something goes wrong.

Reply:
👍